lib/tasks/doc.rake

```diff
@@ -8,69 +8,18 @@ require 'open-uri'
 # Import may benefit from scaling server and database resources
 
 namespace :doc do
-  desc 'Reset database and run all tasks'
-  task reset: [:environment] do
-    Doc::OffenseCode.delete_all
-    Doc::Status.delete_all
-    Doc::Sentence.delete_all
-    Doc::Alias.delete_all
-    Doc::Profile.delete_all
-
-    Importers::Doc::OffenseCode.new('2022-01').perform
-    Importers::Doc::Profile.new('2022-01').perform
-    Importers::Doc::Sentence.new('2022-01').perform
-    Importers::Doc::Alias.new('2022-01').perform
-  end
-
-  desc 'Import all'
-  task :all, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-    Importers::Doc::OffenseCode.new(dir).perform
-    Importers::Doc::Profile.new(dir).perform
-    Importers::Doc::Sentence.new(dir).perform
-    Importers::Doc::Alias.new(dir).perform
-    Importers::Doc::Status.new(dir).perform
-  end
-
-  desc 'Import profiles'
-  task :profiles, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-    Importers::Doc::Profile.new(dir).perform
-  end
-
-  desc 'Import sentences'
-  task :sentences, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-    Importers::Doc::Sentence.new(dir).perform
-  end
-
-  desc 'Import aliases'
-  task :aliases, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-
-    Importers::Doc::Alias.new(dir).perform
-  end
-
-  desc 'Import statutes'
-  task :statutes, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-    Importers::Doc::OffenseCode.new(dir).perform
-  end
-
-  desc 'Import statuses'
-  task :statuses, [:dir] => [:environment] do |_t, args|
-    dir = args.dir
-    Importers::Doc::Status.new(dir).perform
-  end
-
-  desc 'Link to OSCN'
-  task :link_cases, [:county] => [:environment] do |_t, args|
-    county = args.county
-    Importers::Doc::LinkCases.new(county).perform
-  end
-
-  desc 'Link Sentence to OffenseCode'
-  task link_offense_codes: [:environment] do
-    Importers::Doc::LinkOffenseCodes.new.perform
+  desc 'Scrape DOC for most recent file and import'
+  task scrape: [:environment] do
+    dir = '2023-04'
+    Scrapers::Doc::QuarterlyData.perform(dir)
+    ActiveRecord::Base.transaction do
+      Importers::Doc::OffenseCode.new(dir).perform
+      Importers::Doc::Profile.new(dir).perform
+      Importers::Doc::Sentence.new(dir).perform
+      Importers::Doc::Alias.new(dir).perform
+      Importers::Doc::Status.new(dir).perform
+      # Importers::Doc::LinkCases.new(county).perform # uncommented once we add support for multiple counties
+      # Importers::Doc::LinkOffenseCodes.new.perform # uncomment once this is confirmed to be working
+    end
```

Comment:
Is this okay? Removed all the old commands to try to make a clear single entrypoint. If we need to run individual ones can always do it via console or one offs?